go/interpreter/lfvm/instructions_test.go

```diff
@@ -1419,3 +1419,148 @@ func TestGetData(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckSizeOffsetUintOverflow_ReturnsAsExpected(t *testing.T) {
```

Comment:
Nit: 
The name here does not say much. I understand that these are the tests for the function properties:

An improvement would be to make the function TestCheckSizeOffsetUint64Overflow and define the different properties in a table test:
- IgnoresOffsetWithSizeZero
- ReturnsErrorOnSizeOverflow
- ReturnsErrorOnOffsetOverflow
- ReturnsErrorAfterSumOverflow
- ReturnsOkWhenNoOverflow

(I have no preference for CamelCase or normal sentences, I just wish we could unify this some day)

Reply:
ok will work on the rename